src/python/dart/engine/redshift/command/copy.py

```diff
@@ -235,8 +235,12 @@ def _upload_s3_copy_manifests_and_create_tracking_sql_files(action, dataset, dat
         with tempfile.NamedTemporaryFile(delete=False) as f:
             tracking_sql_files.append(f.name)
             schema_name, table_name = get_tracking_schema_and_table_name(action)
-            sql = 'INSERT INTO %s.%s (s3_path, updated) VALUES \n' % (schema_name, table_name)
-            sql += ',\n'.join(["('%s', %s)" % (e[0], "'%s'" % e[1].isoformat() if e[1] else 'NULL') for e in batch])
+            if nudge:
+                sql = 'INSERT INTO %s.%s (s3_path, updated) VALUES \n' % (schema_name, table_name)
+                sql += ',\n'.join(["('%s', %s)" % (e[0], e[1], e[2]) for e in batch])
```

Comment:
It's not very evident what this is doing.

I'd suggest using tuple unpacking:
for s3_uri, created_time, _ in batch